thunder/images/images.py

```diff
@@ -1,5 +1,7 @@
 import logging
 from numpy import ndarray, arange, amax, amin, size, asarray, random, prod
+# Name changes between Python 2 and 3
+from six.moves import xrange
```

Comment:
It seems more "forward looking" to me to just replace all the `xrange`s with `range` and then fix any residual uses of `range` for py2 where necessary (e.g. by wrapping in a list or tuple). Put another way, although `six` makes it easy, it seems backwards to essentially keep using a deprecated API.